react/week1/todo-list/src/todo.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+
+const ToDo = (props) => {
+     
```

Comment:
Better to add some styles to this todo list.

Reply:
Implemented styled